### PR TITLE
Re-grab input when automatic capture is enabled

### LINKED ISSUE #6242: UTM doesn't re-grab input when swiping back and forth in fullscreen.
Even though "capture input automatically when entering full screen" is selected in settings, when I swipe from the fullscreened UTM guest to my host Mac desktop (using three fingers on the trackpad), and then back to the guest, the input does not get re-captured.

**Configuration**  
* UTM Version: Version 4.4.5 (94)
* macOS Version: 14.4.1 (23E224)
* Mac Chip (Intel, M1, ...): M3 Max

### PR DESCRIPTION
**Pull request by haroldm**
Make the window auto-capture the mouse again in full screen when it regains focus (for instance after swiping back to the UTM app).

fixes #6242

### CODE REVIEW

File: Platform/macOS/Display/VMDisplayQemuMetalWindowController.swift [hunk 1]
```diff
@@ -403,19 +403,24 @@ extension VMDisplayQemuMetalWindowController {
     func windowDidEnterFullScreen(_ notification: Notification) {
         isFullScreen = true
         if isFullScreenAutoCapture {
-            captureMouseToolbarButton.state = .on
```
Comment (osy) [inline Platform/macOS/Display/VMDisplayQemuMetalWindowController.swift:406]: Why did you remove this line and 414?

Comment (haroldm) [reply Platform/macOS/Display/VMDisplayQemuMetalWindowController.swift:406]: I think these lines are already present in captureMouse() and releaseMouse(). I just tried to check the behavior but I don't think the toolbar is ever visible in full screen when the mouse is captured.
